app/src/test/java/by/academy/jc/ht/skopinov/ht9/MyStackTest.java

```diff
@@ -0,0 +1,104 @@
+package by.academy.jc.ht.skopinov.ht9;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.LinkedList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class MyStackTest {
```

Comment:
Add unit tests to demonstrate iterator interoperability, I mean, use it in for-each loop and make sure it iterates correctly.